Sources/App/configure.swift

```diff
@@ -22,17 +26,24 @@ public func configure(_ app: Application) async throws {
         openAIKey = key
     }
     
-    // Store API key in application storage for access in routes
+    // Encryption key (optional for development)
+    let encryptionKey: String?
+    if app.environment == .testing {
+        encryptionKey = nil // No encryption in testing
+    } else {
+        encryptionKey = Environment.get("ENCRYPTION_KEY")
```

Comment:
Can we do a sanity check here that they key is a proper key and if not (e.g. parsing it once), just also state that we won't encrypt in that case?